docs/src/targene/data_sources.md

```diff
@@ -144,3 +144,17 @@ Additional optional UK-Biobank files for preprocessing and filtering are:
 
 - `QC_FILE`: A path to the UK-Biobank SNP quality control [`ukb_snp_qc.txt`](https://biobank.ctsu.ox.ac.uk/crystal/refer.cgi?id=1955) file.
 - `UKB_WITHDRAWAL_LIST`: A path to the withdrawal sample list to exclude removed participants from the study.
+
+### All of Us Cohort
+
+TarGene can now be run on the All of Us (AoU) Cohort through the AoU Researcher Workbench. A Workspace must first be created in order to run your analysis-of-interest (See [`Creating a Workspace`](https://support.researchallofus.org/hc/en-us/articles/30143658322836-Creating-a-Workspace)). Your Traits Dataset can then be built on the AoU Researcher Workbench through their interactive Dataset Builder tool (See [`Dataset Builder`](https://support.researchallofus.org/hc/en-us/articles/4556645124244-Building-a-Dataset-with-the-Dataset-Builder) for more information on this). 
+
+The dataset created using these tools must be tailored to pull the traits, confounders and/or covariates-of-interest from the All of Us Cohort, subsetting for samples for which genetic data is available. As TarGene requires both PLINK BED and BGEN files, these must be available for the participants for which trait data is being pulled. 
+
+Using the Dataset Builder tool, you can create a `TRAITS_DATASET` that matches the requirements specific to the `COHORT = "CUSTOM"` mode. This requires your partipicant IDs (named, by default, as `person_id` in AoU) to be contained within a column named `SAMPLE_ID`, and any covariates or confounders to be included in subsequent columns. Each covariate or confounder in the subsequent columns must have a one-to-one mapping with each `SAMPLE_ID`. 
```

Comment:
I don't think I understand this: "Each covariate or confounder in the subsequent columns must have a one-to-one mapping with each `SAMPLE_ID`"

Reply:
This is because there is some longitudinal data. So I am just saying te user must make their own decisions about how to preprocess the data so that there are not multiple measurements of the same trait for a single participant. But I can re-word this to be more easily understandable.